tests/api_internal/test_internal_api_call.py

```diff
@@ -19,28 +19,30 @@
 from __future__ import annotations
```

Comment:
Tests from `tests/api_internal/*` not executed in the CI

Reply:
Good catch. Fix in https://github.com/apache/airflow/pull/28319